app/views/commontator/comments/_show.html.erb

```diff
@@ -137,8 +149,12 @@
                 ) if thread.config.comment_reply_style != :n && !thread.is_closed?
               %>
             </span>
+            <% else %>
+              <span>
+                <a href="#" onclick="return false">Reply</a>
```

Comment:
I think we need to style this link so the colors match the original.

Reply:
Done.